Design-Reviews.md

```diff
@@ -1,26 +1,34 @@
 # Bitcoin Design Reviews
 
-Design reviews are opportunities for bitcoin products to get design critiques on a particular feature or aspect of their product. Actionable suggestions for improving usability, user interface, interaction, and visual aesthetics are given by community members.
+Design reviews are opportunities for designers and product teams to get design feedback on a particular feature or aspect of their product. Actionable suggestions for improving usability, user interface, interaction, and visual aesthetics are given by community members.
 
 ## How does it work?
 
-1. A bitcoin product team submits a page or section they want reviewed in a design reivew  in the [#design-reviews](https://discord.gg/ss2ZbNcv32) channel in our Discord.
+1. A bitcoin product team submits a page or section they want reviewed in a design review in the [#design-reviews](https://discord.gg/ss2ZbNcv32) channel in our Discord.
```

Comment:
1. A bitcoin product or team submits a goal that they have interms of understanding a particular set of user problems or user persona.